win-odbc64/odbcclient/drvr35/cstmt.cpp

```diff
@@ -3389,7 +3389,7 @@ BOOL CStmt::setFetchOutputPerf(long rowsFetched, SQL_DataValue_def*& outputDataV
     m_FetchDataValue.numberOfRows = rowsFetched;
     if(rowsFetched > 0)
     {
-        m_FetchDataValue.rowAddress = new unsigned long[rowsFetched];
+        m_FetchDataValue.rowAddress = new LONG_PTR[rowsFetched];

```

Comment:
Thanks for your prompt response. Use of LONG_PTR is justified very well. No need to change it to void *